content/installation/linux_package_repo/ContrastRepoConfig.md

```diff
@@ -0,0 +1,43 @@
+Some of Contrast's products are distributed via Linux packages. This makes installing and staying up to date very easy. In order to use our package repo you have to configure your package management system to include a reference to our package repo. The instructions for doing this are here.
+
+Some of our packages are tied to the specific Linux distribution you are using. For example, on Ubuntu this could be one of 'bionic', 'xenial', or 'trusty'.  The instructions will show you how to get this information and then use it to configure the right package repo for your system.  This is a one-time configuration.  Once its done, you'll be connected to the software distribution pipeline and can solely manage it via your distribution's package manager.
+
+### Debian/Ubuntu
+
+
+* Configure your system to retrieve from the correct Debian repository. Get the `CODENAME` for your Ubuntu/Debian release. 
+
+```
+grep VERSION_CODENAME /etc/os-release 
```

Comment:
My brain is failing for a fool-proof solution, but could a carefully crafted grep be used to set an ENV variable that could be used in the following script?  e.g.

    export `grep VERSION_CODENAME /etc/os-release`
    curl https://contrastsecurity.jfrog.io/contrastsecurity/api/gpg/key/public | sudo apt-key add -
echo "deb https://contrastsecurity.jfrog.io/contrastsecurity/debian-public/ $VERSION_CODENAME contrast" | sudo tee /etc/apt/sources.list.d/contrast.list

Reply:
I really dislike the whole pkg-repo setup thing.  Its just so 'in-the-weeds' feeling.  I'm considering just making a package that a user downloads directly and sets up the repo for them. Something similar to the epel-release package described here: https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F